graylog2-server/src/main/java/org/graylog2/Configuration.java

```diff
@@ -134,6 +134,12 @@
     @Parameter(value = "dashboard_widget_default_cache_time", validator = PositiveDurationValidator.class)
     private Duration dashboardWidgetDefaultCacheTime = Duration.seconds(10L);
 
+    @Parameter(value = "user_password_default_algorithm")
+    private String userPasswordDefaultAlgorithm = "bcrypt";
+
+    @Parameter(value = "user_password_bcrypt_salt_size")
+    private Integer userPasswordBCryptSaltSize = 10;
```

Comment:
This could be a primitive `int` and should be validated with the `PositiveIntegerValidator`.